Change usage of location label in quickstart (#18)
rancher/elemental-docs: docs/quickstart.md

You can now boot your nodes with this ISO, and they will:

- Register with the registrationURL given and create a per-machine `MachineInventory`
- Install Elemental Teal to the given device
- Reboot

> In order for the `MachineInventorySelectorTemplate` to select the nodes we now need to add a location label to the `MachineInventory`:

Reviewer: ```suggestion
In order for the `MachineInventorySelectorTemplate` to select the nodes, a location label to the `MachineInventory` is now needed:
```
